bastion.yml

```diff
@@ -9,10 +9,10 @@
         mode: 0700
 
   roles:
-    - common
+    - role: common
+      common_hostname: "{{ bonnyci_hostname | default('') }}"
```

Comment:
I dont see bonnyci_hostname being set in any of these commits. Will that end up being set to match ansible_host or inventory hostname?